docs/Displaying a User Interface/Scrollable Message/index.md

```diff
@@ -0,0 +1,140 @@
+# Scrollable Message
+An @![iOS]`SDLScrollableMessage`!@@![android,javaSE,javaEE]`ScrollableMessage`!@ creates a full screen overlay containing a large block of formatted text that can be scrolled. @![iOS]`SDLScrollableMessage`!@@![android,javaSE,javaEE]`ScrollableMessage`!@ contains a body of text, a message timeout, and up to 8 soft buttons depending on head unit. You must check the `DisplayCapabilities` to get the max number of `SoftButtons` allowed by the head unit. 
+
+You simply create  a  @![iOS]`SDLScrollableMessage`!@@![android,javaSE,javaEE]`ScrollableMessage`!@ object and send an RPC request to display the Scrollable Message. 
+
+##  Scrollable Message UI
+![Scrollable Message](assets/ScrollableMessage.png)
+
+@![iOS]
+##### Objective-C
+```objc
+// Create SoftButton Array
+NSMutableArray<SDLSoftButton *> *softButtons = [[NSMutableArray alloc] init];
+
+// Create Message To Display
+NSString *scrollableMessageString = [NSString stringWithFormat:@"Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.\n\n\nVestibulum mattis ullamcorper velit sed ullamcorper morbi tincidunt ornare. Purus in massa tempor nec feugiat nisl pretium fusce id.\n\n\nPharetra convallis posuere morbi leo urna molestie at elementum eu. Dictum sit amet justo donec enim diam."];
+
+// Create a timeout of 50 seconds
+UInt16 scrollableMessageTimeout = 50000;
+
+// Create SoftButtons
+SDLSoftButton *scrollableSoftButton = [[SDLSoftButton alloc] initWithType:SDLSoftButtonTypeText text:@"Button 1" image:nil highlighted:NO buttonId:111 systemAction:nil handler:^(SDLOnButtonPress * _Nullable buttonPress, SDLOnButtonEvent * _Nullable buttonEvent) {
+    if (buttonPress == nil) {
```

Comment:
Make this one line `if (buttonPress == nil) { return; }`

Reply:
fixed